app/controllers/api/v1/owners_controller.rb

```diff
@@ -20,13 +20,13 @@ def create
       ownership = @rubygem.ownerships.new(user: owner, authorizer: @api_key.user)
       if ownership.save
         Delayed::Job.enqueue(OwnershipConfirmationMailer.new(ownership.id))
-        render plain: "#{owner.display_handle} was added as an unconfirmed owner. "\
-                      "Ownership access will be enabled after the user clicks on the confirmation mail sent to their email."
+        render plain: response_with_warnings("#{owner.display_handle} was added as an unconfirmed owner. "\
+                                             "Ownership access will be enabled after the user clicks on the confirmation mail sent to their email.")
```

Comment:
Seems like some stray whitespace has crept in 👻

Reply:
I assume this is an intentional indentation change (but, again, hard to know for sure).

Reply:
I think it's how the diff was rendered when I reviewed. When I saw it the whitespace appeared to be on a line by itself. Looking at the diff fragment now, it's now clearly part of lining up the strings.